eskip/lexer.go

```diff
@@ -131,12 +131,20 @@ func scanEscaped(delimiter byte, code string) ([]byte, string) {
 		c := code[0]
 		isDelimiter := c == delimiter
 		isEscapeChar := c == escapeChar
+		isN := c == 'n'
+		isR := c == 'r'
 
 		if escaped {
-			if !isDelimiter && !isEscapeChar {
+			if !isDelimiter && !isEscapeChar && !isN && !isR {
```

Comment:
can we support the full classic range of:

\b \f \n \r \t \v

?

And in addition, \" which is only a slightly different case

Reply:
Is it only these? Or are there more?

Reply:
Maybe \0 would be nice for some hacks

Reply:
I checked some Go source code and did not see how `\0` was done. So decided to leave that out for now.